### ISSUE TITLE
baked styles refactor - figure/image

### ISSUE THREAD

**Issue by lwtchu** [openstax/cnx-recipes#4630]
### TO DO
- [x] In other parts I'm suggesting to move out figure/image components from them.
- [x] Make sure whether general shape doesn't cover styles for figures in other objects.
- [x] If not, create shapes for them in figure/image files.

Questions
- [x] Caption divider - should we style it everywhere? - LC NOTE: is this the divider between the caption and figure? 
      Edit: It is divider between caption number and caption text. Done within caption refactoring.
- [x] Should we add FigureTopSpacingShape to all themes? - LC NOTE: is this to add spacing at the top of all figures?

### DONE
1. Create new shape for figures from exercises.
  - set `max-width` to 100% for `image`
  - align first media image in ordered list with item
  - align first figure image in ordered list with item
  - align first media image in circled list with item (Corn)
  - create settings for problem, solution, injected question, injected solution
  - apply problem and injected question to all books
  - apply solution and injected solution to books with Answer Key (but not Inline Answer Key)
  
This is how it should look:

![image](https://user-images.githubusercontent.com/40228854/167663508-23719012-9768-4ca0-9b47-2d24848d2253.png)

2. Apply `FigureTopSpacingShape` to all books.

It adds additional space over figure that is right under h4 subtitle:

Before
![image](https://user-images.githubusercontent.com/40228854/167664061-68562909-295b-41b3-be39-d3b05dea7deb.png)

After
![image](https://user-images.githubusercontent.com/40228854/167664171-883bd079-ca1e-4c79-bffc-9442f20b3e11.png)

Oryginally it was added for Political Science (https://github.com/openstax/cnx-recipes/pull/3942)

**Comment by ajluk** [comment]
@lwtchu Do you have any suggestions which books should be tested from each theme for these issue? I guess for sure political science from cosmos theme because it includes the edge case with figure under H4 subtitle, but I'm not sure about the rest.

**Comment by lwtchu** [comment]
@ajluk maybe one from each theme? microbiology, contemporary math, political science, accounting ??

**Comment by lwtchu** [comment]
@ajluk ooh okay let's not do microbio and let's see if we can test college-physics-2e and elementary algebra?

**Comment by ajluk** [comment]
@MartaSztuka please test the regular figures, figures in ordered lists, figures in circled lists (only in math - elementary algebra), in modules, exercises and answer key in College Physics 2e, Elementary Algebra, Political Science, Accounting  and the edge case from description (point 2) (figure that is right under h4 subtitle) from Political Science.

**Comment by MartaSztuka** [comment]
@ajluk there are two problems with figure changes:
1. College Physics 2e
In Answer Key solution number is aligned to the bottom of the picture.
![Screenshot from 2022-05-12 15-31-15.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/a6e175d0-e8e9-43f5-adc9-fa667fb9ae31)
2. Elementary algebra 2e and Accounting
Images that are not figures have smaller top space.
![Screenshot from 2022-05-12 15-51-55.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/c62ba1b9-bd0b-4db1-b030-2abb2686a5a6)
![Screenshot from 2022-05-12 16-01-13.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/e11845b9-faa5-492b-bd7e-52e9a4f7a2bb)
#### PASSED
##### Elementary algebra 2e
![Screenshot from 2022-05-12 15-17-21.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/b24918a6-022a-4972-8feb-1860e53bc627)![Screenshot from 2022-05-12 15-21-35.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/213437ad-c2ef-460a-9c72-2b8281d8eab5)![Screenshot from 2022-05-12 15-20-53.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/ddbff484-de5f-4439-8b4d-8ca94e9f7cd0)
##### College Physics 2e
![Screenshot from 2022-05-12 15-26-18.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/51c17135-d31a-4212-b389-8e31b94b9478)![Screenshot from 2022-05-12 15-28-06.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/18262d0f-e4f3-46d1-9d38-20c3a8d72e19)
![Screenshot from 2022-05-12 15-29-36.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/b3753f0f-534a-427f-a45c-0c0e32e90672)
##### Political Science
![Screenshot from 2022-05-12 15-39-09.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/6ab35f43-3356-499c-be1f-e57ddfc8da4e)
![Screenshot from 2022-05-12 15-35-05.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/49482842-15e2-4b31-97d7-1530f6d8f24a)
##### Financial accounting
![Screenshot from 2022-05-12 16-03-38.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/33f8e01e-cf23-4cad-9c73-b4b087f870c8)
![Screenshot from 2022-05-12 16-04-56.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/b020614e-494b-4e3d-a402-cc5ac672be72)
##### Managerial accounting
![Screenshot from 2022-05-12 15-42-45.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/9e26342d-d6b0-4779-be3c-0ed701c76c13)![Screenshot from 2022-05-12 15-41-49.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/58ae64cc-d11b-48d7-ba95-59090f3a8842)

**Comment by ajluk** [comment]
@lwtchu I have checked and points 1 and 2 are not a regression. These images looked like that before the refactoring changes: 
1. College Physics 2e - this will require recipe fix for supporting bake first elements in exercises/solutions
![image](https://user-images.githubusercontent.com/55203598/168136664-ea935659-1a96-424b-9591-2d365917ec0c.png) Do we want to open a separate issue for it?
2. Accounting - This is not a bug, we removed margins from paragraphs in exercises within this issue: https://github.com/openstax/cnx-recipes/issues/3330. Should it stay this way?
![image](https://user-images.githubusercontent.com/55203598/168137244-2c5a4ee3-3d07-44e0-9f00-6150b93372dc.png)
